vms/avm/vm_benchmark_test.go

```diff
@@ -52,7 +55,7 @@ func BenchmarkLoadUser(b *testing.B) {
 		require.NoError(user.Close())
 	}
 
-	benchmarkSize := []int{10, 100, 1000, 10000}
+	benchmarkSize := []int{10, 100, 1000, 5000}
```

Comment:
reasonable change

Reply:
The maximum amount allowed here is actually `5000`. The benchmark for `10000` was failing on dev.